_posts/2020-07-15-#19031.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: pr
+date: 2020-07-15
+title: "Implement ADDRv2 support (part of BIP155)"
+pr: 19031
+authors: [vasild]
+components: ["p2p"]
+host: troygiorshev
+status: upcoming
+commit:
+---
+
+## Notes
+
+  * [PR 19031](https://github.com/bitcoin/bitcoin/pull/19031) is the implementation of [BIP 155](https://github.com/bitcoin/bips/blob/master/bip-0155.mediawiki), the `addrv2` message.
+  * The smaller PR that's currently being worked on, and should be reviewed first, is [PR 19360](https://github.com/bitcoin/bitcoin/pull/19360).
+  * Ultimately this will allow us to work with [Tor v3](https://github.com/bitcoin/bitcoin/issues/18884), along with many other different types of networks.
```

Comment:
If you have time, don't hesitate to expand this section by providing more resources in the notes: more background, context, info on the topic. See sessions like:

- https://bitcoincore.reviews/15681
- https://bitcoincore.reviews/16702
- https://bitcoincore.reviews/17639

Reply:
I agree. A few words about what addr v2 and tor v3 are would be helpful.